registry/zookeeper/registry_test.go

```diff
@@ -74,6 +99,39 @@ func Test_Subscribe(t *testing.T) {
 	defer ts.Stop()
 }
 
+func Test_NoSubscribe(t *testing.T) {
+	regurl, _ := common.NewURL("registry://127.0.0.1:1111", common.WithParamsValue(constant.ROLE_KEY, strconv.Itoa(common.PROVIDER)))
+	url, _ := common.NewURL("dubbo://127.0.0.1:20000/com.ikurento.user.UserProvider", common.WithParamsValue(constant.CLUSTER_KEY, "mock"), common.WithMethods([]string{"GetUser", "AddUser"}))
+	ts, reg, _ := newMockZkRegistry(&regurl)
+
+	//provider register
+	err := reg.Register(url)
+	assert.NoError(t, err)
+
+	if err != nil {
```

Comment:
why add err != nil? by `assert.NoError(t, err)`, when err != nil, the ut will fail.

Reply:
This block is the same to Test_subsribe ,or you mean Test_subscribe is incorrect too ?

https://github.com/apache/dubbo-go/blob/8a3c02c9b7737a12712451b16a3cdc8ce46a5c7d/registry/zookeeper/registry_test.go#L57

Reply:
it is not necessary. i think it is better to delete them. or other resons?

Reply:
@hxmhlt can you remember the reason? If not , can i delete it?